ui/apps/platform/src/ConsolePlugin/PluginProvider.tsx

```diff
@@ -2,9 +2,11 @@ import React, { useMemo, type ReactNode } from 'react';
 import { ApolloProvider } from '@apollo/client';
 
 import axios from 'services/instance';
+import { UserPermissionProvider } from 'providers/UserPermissionProvider';
+import { FeatureFlagsProvider } from 'providers/FeatureFlagProvider';
```

Comment:
This pattern for providers puts you in good company :)
https://humanwhocodes.com/blog/2019/01/stop-using-default-exports-javascript-module/